airmozilla/popcorn/static/popcorn/js/editor.js

```diff
@@ -17,4 +17,20 @@ $(document).ready(function () {
   })
   // Initialize the editor with the div id and path to Popcorn Editor.
   PopcornEditor.init('editor', '/static/popcorn/PopcornEditor/editor.html');
+  PopcornEditor.listen('save', function (message) {
+    console.log(message);
+    $.post($('#editor').data('save'), {
+      data: JSON.stringify(message.data),
+      slug: $('#editor').data('slug'),
+      csrfmiddlewaretoken: $('input[name="csrfmiddlewaretoken"]').val()
+    })
+    .done(function (response) {
+      alert('Event is saved and processing with PopcornEdit.id = '
+           + response.id);
```

Comment:
Sure, I'm fine with updating it to that but do we plan on any mortals even being able to see this before we implement the modal?